pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -71,22 +74,27 @@ var (
 	AntreaInfoElementsIPv6 = append(antreaInfoElementsCommon, []string{"destinationClusterIPv6"}...)
 )
 
+const (
+	idleTimeoutReason   = uint8(0x01)
+	activeTimeoutReason = uint8(0x02)
+)
```

Comment:
one would think these would be defined in the go-ipfix library?

Reply:
yes good idea.. will try to add this new patch release